skimage/filters/_gaussian.py

```diff
@@ -19,8 +19,8 @@ def _convert_input(image, preserve_range):
 
 
 def gaussian(image, sigma=1, output=None, mode='nearest', cval=0,
-             multichannel=None, preserve_range=False):
-    """Multi-dimensional Gaussian filter
+             multichannel=None, preserve_range=False, truncate=4.0):
```

Comment:
@soupault We can have this discussion here #2422 if you want :) I'm open to any suggestion.